lib/senders/index.js

```diff
@@ -120,15 +120,44 @@ module.exports = function (log, config, error, db, translator, sender) {
           })
       },
       sendVerifyLoginEmail: function (account, code, opts) {
+        var mailerPromises = account.emails.map(function (emailRecord) {
+          if (emailRecord.isPrimary || emailRecord.isVerified) {
+            return getSafeMailer(emailRecord.email)
+              .then(function (mailer) {
+                return mailer.verifyLoginEmail({
+                  acceptLanguage: opts.acceptLanguage || defaultLanguage,
+                  code: code.toString('hex'),
+                  email: emailRecord.email,
+                  ip: opts.ip,
+                  flowId: opts.flowId,
+                  flowBeginTime: opts.flowBeginTime,
+                  location: opts.location,
+                  redirectTo: opts.redirectTo,
+                  resume: opts.resume,
+                  service: opts.service,
+                  timeZone: opts.timeZone,
+                  uaBrowser: opts.uaBrowser,
+                  uaBrowserVersion: opts.uaBrowserVersion,
+                  uaOS: opts.uaOS,
+                  uaOSVersion: opts.uaOSVersion,
+                  uid: account.uid.toString('hex')
+                })
+              })
+          } else {
+            P.resolve()
+          }
+        })
+
+        return P.all(mailerPromises)
+      },
+      sendVerifySecondaryEmail: function (account, code, opts) {
         return getSafeMailer(account.email)
           .then(function (mailer) {
-            return mailer.verifyLoginEmail({
+            return mailer.verifySecondaryEmail({
               acceptLanguage: opts.acceptLanguage || defaultLanguage,
               code: code.toString('hex'),
-              email: account.email,
+              email: opts.email,
```

Comment:
This diff highlights the inconsistency here - both `sendVerifySecondaryEmail(account, code, opts)` and `sendVerifyLoginEmail(account, code, opts)` have exactly the same call signature, but the former sends to all entries in `account.emails` and the later sends to a single address from `opts.email`.  Seems like real footgun potential.